benchmarks/schema.gql

```diff
@@ -0,0 +1,42 @@
+type Query {
+        people_dataclass: [Person!]!
```

Comment:
Don't use spearate `_dataclass` fields. We can use `root_value` to pass either raw or hydrated data to `GraphQL()` app for default resolvers to use.

Reply:
As above, we have only one query now and we pass data by root_value